### PR TITLE
feat: email template with more content

### PR DESCRIPTION
**Pull request by adithyanmadhi**
Added a new task3.html page
Screenshot 2024-01-08 005915

Screenshot 2024-01-08 010616
Screenshot 2024-01-08 010038

JIRA : 
https://mahendrankannan.atlassian.net/browse/ON-14

### CODE REVIEW

File: email-templates/task1.html
Comment (gkmahendran09) [inline email-templates/task1.html:1]: Delete this file

File: email-templates/task2.html
Comment (gkmahendran09) [inline email-templates/task2.html:1]: Delete this file

File: email-templates/task3.html
Comment (gkmahendran09) [inline email-templates/task3.html:1]: Use HTML formatter to format the document - https://code.visualstudio.com/docs/languages/html

File: email-templates/task3.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
...own for his exceptional dribbling skills, incredible speed, and his exceptional goalscoring ability.Cristiano Ronaldo dos Santos Aveiro GOIH ComM (Portuguese pronunciation: [kɾiʃˈtjɐnu ʁɔˈnaldu]; born 5 February 1985) is a Portuguese professional footballer who plays as a forward for and captains both Saudi Pro League club Al Nassr and the Portugal national team. Widely regarded as one of the greatest players of all-time, Ronaldo has won five Ballon d'Or awards,[note 3] a record three UEFA Men's Player of the Year Awards, and four European Golden Shoes, the most by a European player. He has won 34 trophies in his career, including seven league titles, five UEFA Champions Leagues, the UEFA European Championship and the UEFA Nations League. Ronaldo holds the records for most appearances (183), goals (140) and assists (42) in the Champions League, goals in the European Championship (14), international goals (128) and international appearances (205). He is one of the few players to have made over 1,200 professional career appearances, the most by an outfield player, and has scored over 850 official senior career goals for club and country, making him the top goalscorer of all time.</P>
```
Comment (gkmahendran09) [inline email-templates/task3.html:58]: Don't use capital letters for any tag always use lowercase like `<p>`.

File: email-templates/task3.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+      </head>
+<body>
+    
+    <div class="container">
+        <header>
+            <h1>CRISTIANO RONALDO</h1>
+          </header>
+        
+     <div class="body">
+        <i>
```
Comment (gkmahendran09) [inline email-templates/task3.html:57]: The order tag should be descending like `div > p > i/b/u`

**Review by gkmahendran09 — CHANGES_REQUESTED** [review]
General comments:

- Update the JIRA link with the absolute link in the PR description and refer to the previous PR and update any other feedbacks

File: email-templates/task3.html [hunk 3]
```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <style media="screen">
```
Comment (gkmahendran09) [inline email-templates/task3.html:6]: no need to use `media` keyword.

File: email-templates/task3.html [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+         .logo{text-align: center;
+         padding-bottom: 20px;
+         }
+         .main-content {
+         text-align: center;
+         padding: 20px;
+         text-align: center;
+         }
+         img{overflow: visible;
+         }
```
Comment (gkmahendran09) [inline email-templates/task3.html:40]: CSS is not formatted correctly. Please check and correct it

**Review by gkmahendran09 — DISMISSED** [review]
Fix the CSS formating and then merge it ( use squash and merge while merging )

File: email-templates/task3.html (hunk 3 above)
Comment (adithyanmadhi) [reply email-templates/task3.html:6]: removed the media Keyword

File: email-templates/task3.html (hunk 4 above)
Comment (adithyanmadhi) [reply email-templates/task3.html]: corrected the  CSS part.

File: email-templates/task3.html (hunk 2 above)
Comment (adithyanmadhi) [reply email-templates/task3.html]: re arranged in the order

**Review by gkmahendran09 — DISMISSED** [review]

**Review by gkmahendran09 — APPROVED** [review]

**Review by gkmahendran09 — APPROVED** [review]
Look good.

**Review by gkmahendran09 — APPROVED** [review]

**Pull request merged**